### ISSUE TITLE
Common way of using `pkg-dir`

### ISSUE THREAD

**Issue by rauschma** [sindresorhus/pkg-dir#16]
One common way of using this package is as follows. Maybe mention that in the readme?

```js
import {packageDirectorySync} from 'pkg-dir';
console.log(packageDirectorySync({cwd: import.meta.url}));
```

* URLs in strings seem to work, I didn’t have to use `{cwd: new URL(import.meta.url)}`.
* If I’m not mistaken, using the default value `process.cwd()` for `cwd` can produce incorrect results for modules.

**Comment by sindresorhus** [comment]
> One common way of using this package is as follows. Maybe mention that in the readme?

Yes. Good idea. Although, I prefer to just add official URL support. I'm slowly adding URL support to places that accept paths.

> If I’m not mistaken, using the default value process.cwd() for cwd can produce incorrect results for modules.

It really depends on your use-case.

**Comment by rauschma** [comment]
> I prefer to just add official URL support

So the safe way to make the call will eventually be the following code(?)

```js
packageDirectorySync({cwd: new URL(import.meta.url)})
```

> It really depends on your use-case.

True!

**Comment by sindresorhus** [comment]
> So the safe way to make the call will eventually be the following code(?)

Correct. Hopefully, Node.js will eventually get a nicer API to get the current file and directory.
